test/library/packages/ZMQ/pair.chpl

```diff
@@ -0,0 +1,131 @@
+use Help;
```

Comment:
This needs an expected output file, see https://github.com/chapel-lang/chapel/blob/master/doc/rst/developer/bestPractices/TestSystem.rst#a-correctness-test